CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 2.6.0 (Unreleased)
 ==================
+- [NEW] Added ``Cloudant.bluemix()`` client class method for instantiating a Cloudant Bluemix client.
```

Comment:
I'm not sure a "Cloudant Bluemix client" is really a thing.
Instantiating a client from VCAP_SERVICES metadata?
Last time we went with:
> - [NEW] Added ``cloudant_bluemix`` method to the Cloudant client allowing service credentials to be passed using the CloudFoundry VCAP_SERVICES environment variable.